network-api/networkapi/wagtailpages/models.py

```diff
@@ -165,6 +165,7 @@ class MiniSiteNameSpace(ModularPage):
         'CampaignPage',
         'OpportunityPage',
         'BlogPage',
+        'SpecialTemplatePage',
```

Comment:
I'm pretty sure we don't have much of a need for MiniSiteNameSpace pages anymore; Let's make this a subpage type of indexPage and maybe Homepage

Reply:
Def. an option - the IndexPage currently does not restrict what its children can be, so should already allow this.